spatial-tracking-explainer.md

```diff
@@ -364,7 +403,9 @@ dictionary XRReferenceSpaceOptions {
   required XRReferenceSpaceType type;
 };
 
-[SecureContext, Exposed=Window] interface XRReferenceSpace : XRSpace {
+[SecureContext, Exposed=Window] interface XRReferenceSpace : XRSpace {  
+  attribute XRRigidTransform? originTransform;
```

Comment:
Is there usefulness in this being nullable?  Or can we get away with just making it be identity by default?

Reply:
Seems reasonable to me!